### PR TITLE
implemented ExchangeSort under sort alogorithms

### PR DESCRIPTION
**Pull request by absharul**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

### CODE REVIEW

**Comment by absharul** [comment]
Implemented ExchangeSort algorithm

File: src/main/java/com/thealgorithms/sorts/ExchangeSort.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package com.thealgorithms.sorts;
```
Comment (absharul) [inline src/main/java/com/thealgorithms/sorts/ExchangeSort.java:1]:
```suggestion
package com.thealgorithms.sorts;
```

**Comment by BamaCharanChhandogi** [comment]
solve clang issue. and then add the test cases file.

**Comment by absharul** [comment]
> solve clang issue. and then add the test cases file.

Hey BamaCharanChhandogi I'm pleased to inform you that I have successfully resolved the Clang issue.

**Comment by absharul** [comment]
would you please check this PR??

**Review by BamaCharanChhandogi — CHANGES_REQUESTED** [review]
Remove the main method, then add a test case file.

**Pull request closed without merging**